contracts/src/Gateway.sol

```diff
@@ -224,6 +224,14 @@ contract Gateway is IGateway, IInitializable {
     /**
      * Getters
      */
+    function beefyClient() external view returns (address) {
+        return BEEFY_CLIENT;
+    }
+
+    function agentExecutor() external view returns (address) {
+        return AGENT_EXECUTOR;
+    }
+
```

Comment:
Not necessary. If a variable or constant is public it automatically has an accessor generated.

Reply:
addressed in https://github.com/Snowfork/snowbridge/pull/1167/commits/022ab0e88e1fbc24fb507874b435c28ceea85252